extensions/wikia/ThemeDesigner/ThemeSettings.class.php

```diff
@@ -28,8 +28,12 @@ class ThemeSettings {
 	const FaviconImageName = 'Favicon.ico';
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this constant "FaviconImageName" to match the regular expression ^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS115)